Bulk update empty

lorawoodford opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->
When using the bulk update options for top containers, bulk updating to empty values (e.g. leaving the text or selection fields empty) does remove these values from the records in the database.  However, in the case of locations (either bulk updating single or multiple locations) while the database is updated, the top container record is not reindexed so results in the manage top containers screen do not reflect the reality of the database.  This resolves that issue and ensures that all changes made with bulk update location(s) trigger a reindex.

## Description
<!--- Describe your changes in detail -->
Updated `backend/app/model/top_container.rb` to ensure `update_mtime_for_ids` is run even when an empty `location_uri` is provided.

Also, to complement this work:

- updated .yml files updated to add clarifying text noting that providing empty ils_holdings, container_profiles, or locations will remove currently linked/provided values;
- modified `top_containers.bulk.js` to resize the confirmation modal on bulk actions so it does not fill the entire screen height;
- added `btn-default` style to several cancel/confirmation buttons to make it more visually obvious these are clickable buttons; 
- added two new tests; and,
- updated several tests in `model_managed_containers_spec.rb` that included unused variables. 

## Related JIRA Ticket or GitHub Issue
<!--- Please link to the JIRA Ticket or GitHub Issue here: -->

## Motivation and Context
<!--- Why is this change required? What problem does it solve? -->
Corrects inconsistency between index state and db state.

## How Has This Been Tested?
<!--- Please describe in detail how you tested your changes. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->
Added two new tests to `model_managed_containers_spec.rb` to confirm passing empty values to the location_uri updates both the db and the index.

## Screenshots (if appropriate):
Changes to help text:
<img width="1145" alt="top_container_indexing_language" src="https://user-images.githubusercontent.com/15144646/69662781-00349f00-1053-11ea-8cfe-158c4b145b80.png">

Resizing the confirmation modal:
<img width="1141" alt="top_container_smaller_modal" src="https://user-images.githubusercontent.com/15144646/69662813-0b87ca80-1053-11ea-88e5-6631acc7c0eb.png">


## Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

lorawoodford commented:
This should resolve https://archivesspace.atlassian.net/browse/ANW-973.  I didn't see that ticket until after this PR was submitted. :-)

lmcglohon commented on backend/app/model/top_container.rb:
```diff
@@ -348,22 +348,24 @@ def self.bulk_update_location(ids, location_uri)
       relationship.handle_delete(relationship.find_by_participant_ids(TopContainer, ids).select{|v| v.status == 'current'}.map(&:id))
 
-      location = Location[JSONModel(:location).id_for(location_uri)]
+      unless location_uri.empty?
```
@lorawoodford I wonder if you shouldn't perform a strip on location_uri in case someone uses the space bar to clear out the location causing the location_uri to be one or more blank spaces?

lorawoodford replied:
@lmcglohon - As we discussed, this isn't necessary here, but I did go add it to the batch update that handles ILS holdings information as that text field could contain nothing but whitespace.

lorawoodford commented:
Added the strip for ILS holdings information, and added help text for rapid barcode update that clarifies that supplying empty/blank barcodes will remove any existing barcodes from the top container record(s).
